controllers/onboardingExtension.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  ERROR_WHILE_CREATING_REQUEST,
+  LOG_ACTION,
+  ONBOARDING_REQUEST_CREATED_SUCCESSFULLY,
+  REQUEST_ALREADY_PENDING,
+  REQUEST_LOG_TYPE,
+  REQUEST_STATE,
+  REQUEST_TYPE,
+  UNAUTHORIZED_TO_CREATE_ONBOARDING_EXTENSION_REQUEST,
+} from "../constants/requests";
+import { userState } from "../constants/userStatus";
+import { addLog } from "../models/logs";
+import { createRequest, getRequestByKeyValues } from "../models/requests";
+import { fetchUser } from "../models/users";
+import { getUserStatus } from "../models/userStatus";
+import { User } from "../typeDefinitions/users";
+import { OnboardingExtension, OnboardingExtensionCreateRequest, OnboardingExtensionResponse } from "../types/onboardingExtension";
+
+export const createOnboardingExtensionRequestController = async (req: OnboardingExtensionCreateRequest, res: OnboardingExtensionResponse) => {
```

Comment:
can we please write a jsDOC comment for this new function?

Reply:
Resolved